backend/amd64/emit.mlp

```diff
@@ -1582,9 +1642,12 @@ let emit_instr fallthrough i =
       I.bswap (res32 i 0);
   | Lop(Ispecific(Ibswap { bitwidth = Sixtyfour })) ->
       I.bswap (res i 0)
-  | Lop(Ispecific(Ifloatsqrtf addr)) ->
+  | Lop(Ispecific(Ifloatsqrtf (Float64, addr))) ->
       I.xorpd (res i 0) (res i 0); (* avoid partial register stall *)
       I.sqrtsd (addressing addr REAL8 i 0) (res i 0)
+  | Lop(Ispecific(Ifloatsqrtf (Float32, addr))) ->
+      I.xorps (res i 0) (res i 0); (* avoid partial register stall *)
+      I.sqrtss (addressing addr REAL8 i 0) (res i 0)
```

Comment:
Not sure why it is the same as the
`Float64` case; shouldn't we address
a 32-bit value?
(Also, is this actually generated?)

Reply:
This should use `REAL4`, yeah. 
It's not currently being generated because I didn't add an override in selection. 
I've just made this a failure case; I will add an SSE2 instruction `Sqrt_scalar_f32` in another PR.